Exercises/if-else-switch/if-else-switch.js

```diff
@@ -6,9 +6,44 @@ const codeToMissingComplexityElement = {
 };
 
 function getMissingComplexityElementCode(password) {
+  if (!password) {
+    return codeToMissingComplexityElement[3];
+  }
+  if (password.length <= 8) {
+    return codeToMissingComplexityElement[0];
+  }
+  if (password === password.toLowerCase()) {
+    return codeToMissingComplexityElement[1];
+  }
+  if (!password.match(/.*[^A-Za-z0-9]/)) {
+    return codeToMissingComplexityElement[2];
+  }
+  return 'Good job!';
+}
 
+function getErrorMessage(getMissingComplexityElementCode) {
+  // eslint-disable-next-line default-case
+  switch (getMissingComplexityElementCode) {
```

Comment:
you got code, and you need now using mapping above get missed element, and generate accordingly message. Imagine that eventually code of length becomes 1

Reply:
you are duplicating yourself, and hardcode few times, value: 
```
  case 0:
    console.log(codeToMissingComplexityElement[0]);
    break;
```
pls think how to avoid it